README.md

```diff
@@ -97,17 +97,26 @@ The server interface defined in `gen-go/server/interface.go` has one method for
 
 
 ### Logging
-  The [kayvee middleware logger](https://godoc.org/gopkg.in/Clever/kayvee-go.v5/middleware) is automatically added to the context object.
+  The [kayvee middleware logger](https://godoc.org/gopkg.in/Clever/kayvee-go.v6/middleware) is automatically added to the context object.
   It can be pulled out of the context object and used via the kayvee `FromContext` method:
 
 ```go
-import "gopkg.in/Clever/kayvee-go.v5/logger"
+import "gopkg.in/Clever/kayvee-go.v6/logger"
 ...
 logger.FromContext(ctx).Info(...)
 ```
 
   You should use this logger for all logging within your controller implementation.
 
+#### Application Log Routing
```

Comment:
this repo is public, so i wonder if we should discuss log routing without making it clear it's not clever-specific.

should we also add this to: https://github.com/Clever/template-wag with an empty kv config? Doesn't necessarily need to be you who makes the change